src/Microsoft.Health.Fhir.Azure/ExportDestinationClient/AzureAccessTokenClientInitializer.cs

```diff
@@ -7,42 +7,37 @@
 using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Storage.Blobs;
 using EnsureThat;
-using Microsoft.Azure.Storage.Auth;
-using Microsoft.Azure.Storage.Blob;
-using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Microsoft.Health.Fhir.Core.Configs;
 using Microsoft.Health.Fhir.Core.Features.Operations.Export.ExportDestinationClient;
 
 namespace Microsoft.Health.Fhir.Azure.ExportDestinationClient
 {
-    public class AzureAccessTokenClientInitializer : IExportClientInitializer<CloudBlobClient>
+    public class AzureAccessTokenClientInitializer : IExportClientInitializer<BlobServiceClient>
     {
-        private readonly IAccessTokenProvider _accessTokenProvider;
+        private readonly TokenCredential _credential;
         private readonly ExportJobConfiguration _exportJobConfiguration;
-        private readonly ILogger<AzureAccessTokenClientInitializer> _logger;
 
         public AzureAccessTokenClientInitializer(
-            IAccessTokenProvider accessTokenProvider,
-            IOptions<ExportJobConfiguration> exportJobConfiguration,
-            ILogger<AzureAccessTokenClientInitializer> logger)
+            TokenCredential credential,
```

Comment:
Since we do not register a service of type `TokenCredential` the framework will fail to resolve it and hence cannot create an `AzureAccessTokenClientInitializer` object.

Reply:
Good catch! I forgot to include that change in the PR.